### ISSUE TITLE
Windows EXE -  ImportError: attempted relative import with no known parent package

### ISSUE THREAD

**Issue by dutchcolonial** [philosowaffle/peloton-to-garmin#125]
Installed the windows version of the product.
Put both the program and the data directories in a directory called PelotonToGarm under My Documents (to make sure I always have write access"
Configured the json file, put the right username and pwd in for both Garmin and Peloton (tested both).
Ran the PelotonToGarminConsole.exe program
Program downloads the latests workout from Peloton (fit format).
Program tries to upload the data but then I get an error:
[09:49:20 ERR] Failed to upload workouts. GUpload had exit code of 1
[09:49:20 ERR] GUpload returned an error code. Failed to upload workouts.
Garmin.GarminUploadException: GUpload Exit Code: 1 - GUpload returned an error code. Failed to upload workouts.
   at Garmin.GarminUploader.UploadViaPython(String[] files) in D:\a\peloton-to-garmin\peloton-to-garmin\src\Garmin\GarminUploader.cs:line 117
   at Garmin.GarminUploader.UploadToGarmin() in D:\a\peloton-to-garmin\peloton-to-garmin\src\Garmin\GarminUploader.cs:line 54
   at PelotonToGarminConsole.Program.RunAsync(Configuration config)

I have no idea where the d:\a\peloton..... path is coming from.

OS: WIndows 10 64bit
Method: EXE
Version V2.0.1

Log file attached.

[log20210621.txt](https://github.com/philosowaffle/peloton-to-garmin/files/6687078/log20210621.txt)

**Comment by philosowaffle** [comment]
@dutchcolonial think I have a fix for this. Can you [download this version](https://github.com/philosowaffle/peloton-to-garmin/suites/3048010751/artifacts/69267073) and confirm how it works. Note, it may still fail on the Garmin upload step if the Authentication fails (#109 ), but the original `Import Error` should be resolved now.

**Comment by dutchcolonial** [comment]
Installed the update. The error is gone (no errors at all). However, I don't thing any upload as attempted (see attached log).
[log20210621.txt](https://github.com/philosowaffle/peloton-to-garmin/files/6688480/log20210621.txt)

**Comment by philosowaffle** [comment]
hmmmm, can you double check your config file still has Garmin upload enabled?

**Comment by dutchcolonial** [comment]
Oops, blush, my bad. Changed that and cleared out the output directory. Ran the tool again (see log attached). Get error messages now for the upload. Wondering if it is  path name related (see / and \ in the same path). My config file entries only have forward slashes (/).
I checked the upload directory and it actually has the .fit file in it.
[log20210621.txt](https://github.com/philosowaffle/peloton-to-garmin/files/6688920/log20210621.txt)
